Survey Form

```diff
@@ -1,35 +1,83 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-<title>Survey Form</title>
-</head>
-<body>
-    <h1 id= "title">Survey Form</h1>
-    <p id= "description">This is my survey form</p>
-    <form id="survey-form">
-      <label  id="name-label">
-<input type="text" id="name" placeholder="name" required="required ">Name</label >
-<label  id="email-label">
-<input type="email" id="email" required="required " placeholder="Email">Email</label >
-<label  id="number-label">
-<input type="number" id="number" placeholder="Number" min="10" max="20">Number</label >
-<select id="dropdown">
-  <option value="">Male
-    </option>
-    <option value="">Female
-    </option>
-    </select>
-    <input type="radio" value="male" name="white">
-    <input type="radio" value="male" name="white">
-    </input>
-    </input>
-    <input type="radio" value="Female" name="black">
-    <input type="radio" value="Female" name="black"></input>
-    </input>
-  <input type="checkbox" value="check1"></input>
-  <input type="checkbox" value="check1"></input>
-  <textarea type="textarea" id="survey-form" placeholder="comment here"></textarea>
-  <input type="submit" id="submit" value="Submit">
-    </form>
-</body>
+<!DOCTYPE.html> 
+<html>
+    <header>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Input Form</title>
+        <link rel="stylesheet" href="inputForm.css">
+    </header>
```

Comment:
Here I have used incorrect tags, I should have used >head tags.

Reply:
Please, note that this activity requires that you perform a code review and make comments with your suggested changes as a code reviewer. For example, the above comment as a review comment could be:

"Kindly use the `<head>` tag instead of the `<header>` tag. While The head tag is used for holding Meta information, title, links, etc. and is not displayed on the page, the header tag is used within the body of the website and can be used multiple times if required, e.g. to determine the top of an article".

Kindly modify your comments to take the form of reviews. Remember to give reasons for the suggested change.